model/server_settings.go

```diff
@@ -117,6 +118,18 @@ type LoginWith struct {
 	Federated bool `yaml:"federated" json:"federated,omitempty"`
 }
 
+// TFAType is a type of two-factor authentication for apps that support it.
+type TFAType string
+
+const (
+	// TFATypeApp is an app (like Google Authenticator).
+	TFATypeApp TFAType = "app"
+	// TFATypeSMS is an SMS.
+	TFATypeSMS TFAType = "sms"
```

Comment:
SMS is a transport, we will extend it with Push notifications later

Reply:
That exactly what I meant - to describe different types of transport, since the underlying algo is the same.